### PR TITLE
Made collections info panel compact for mobile/smaller views

### PR DESCRIPTION
**Pull request by karthikshetty03**
Used media Queries in collections-info-overlay-component.scss file
Added EventListener for screen resizing in collections-info-component.ts file
Changes in smallScreen layout in collections-info-component.html file

### CODE REVIEW

File: packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.ts [hunk 1]
```diff
@@ -3,58 +3,72 @@ import { PrettySymbols } from 'phoenix-event-display';
 import { EventDisplayService } from '../../../../services/event-display.service';
 
 @Component({
-  selector: 'app-collections-info-overlay',
-  templateUrl: './collections-info-overlay.component.html',
-  styleUrls: ['./collections-info-overlay.component.scss']
+	selector: 'app-collections-info-overlay',
```
Comment (9inpachi) [inline packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.ts:6]: For now, please configure your editor to use 2 spaces for tabs and format the TypeScript code. I intend to add prettier later but for now you will need to do this manually.

File: package.json [hunk 2]
```diff
@@ -15,5 +15,8 @@
   },
   "devDependencies": {
     "lerna": "^3.22.1"
+  },
+  "dependencies": {
+    "yarn": "^1.22.10"
```
[the remark this answers is not in the document]
Comment (karthikshetty03) [reply package.json:20]: Noted and made necessary changes.

File: packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.scss [hunk 3]
```diff
@@ -122,3 +114,55 @@
...
+
+    &:last-child {
+      padding-right: 0;
+    }
+  }
+}
+
+  
+
+
```
[the remark this answers is not in the document]
Comment (karthikshetty03) [reply packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.scss]: Noted.

File: packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.ts (hunk 1 above)
Comment (karthikshetty03) [reply packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.ts]: Reverted and inserted the part of code into the original .ts file
Thanks a lot for the feedbacks!

**Comment by karthikshetty03** [comment]
@9inpachi Can you please review the PR? I have made relevant changes now.

File: packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.scss [hunk 4]
```diff
@@ -122,3 +114,50 @@
...
+  .table-responsive {
+    overflow-x: hidden;
+  }
+  #collectionTable {
+    td, thead tr th {
+      display : block;
+    }
+  }
+  mat-divider {
+    border: 1px solid white;
```
[the remark this answers is not in the document]
Comment (karthikshetty03) [reply packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.scss:149]: Ohh, did not realize this. Noted and fixed.

File: packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.ts [hunk 5]
```diff
@@ -14,11 +14,23 @@ export class CollectionsInfoOverlayComponent implements OnInit {
...
+		let screenSize = window.matchMedia('(max-width: 600px)');
+		screenSize.matches == true ? (this.isSmallScreen = true) : (this.isSmallScreen = false);
+		screenSize.addEventListener('change', () => {
+			if (screenSize.matches == true) {
+				this.isSmallScreen = true;
+			} else {
+				this.isSmallScreen = false;
+			}
+		});
+
```
[the remark this answers is not in the document]
Comment (karthikshetty03) [reply packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.ts:33]: Got it. Thanks!

File: packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.html [hunk 6]
```diff
@@ -14,10 +14,9 @@
         </select>
       </div>
     </div>
-
     <div class="boxBody table-responsive">
       <table id="collectionTable" class="table table-borderless table-sm" *ngIf="showingCollection">
-        <thead>
+        <thead *ngIf="!isSmallScreen">
```
[the remark this answers is not in the document]
Comment (karthikshetty03) [reply packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.html:19]: Yes, we can do this without using js and only CSS media queries.
Noted and fixed.

File: packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info.component.ts [hunk 7]
```diff
@@ -4,30 +4,29 @@ import { ComponentPortal } from '@angular/cdk/portal';
...
-  }
-
-  toggleOverlay() {
-    this.showObjectsInfo = !this.showObjectsInfo;
-    this.overlayWindow.instance.showObjectsInfo = this.showObjectsInfo;
-  }
+	toggleOverlay() {
+		this.showObjectsInfo = !this.showObjectsInfo;
+		this.overlayWindow.instance.showObjectsInfo = this.showObjectsInfo;
+	}
```
[the remark this answers is not in the document]
Comment (karthikshetty03) [reply packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info.component.ts:31]: I am facing this problem of diff, like it shows up even if I have not made any changes to it.
Looking for a workaround for this.

File: packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.scss [hunk 8]
```diff
@@ -122,3 +114,50 @@
...
+    display: block;
+    margin: 10px; 
+  }
+}
+
+@media (min-width:600px)  {
+  tr * {
+    padding-right: 1.2rem;
```
[the remark this answers is not in the document]
Comment (karthikshetty03) [reply packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info-overlay/collections-info-overlay.component.scss:157]: Yes, it is required for correct alignment (proper spacing in the table) on large screen views.
Actually, it was present before; I just put it in a media query to apply only on large screens.

File: packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info.component.ts (hunk 7 above)
[the remark this answers is not in the document]
Comment (9inpachi) [reply packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info.component.ts]: I think you made the changes at some point and then manually tried to revert those.

I think your code editor is configured to indent with tabs of 2 spaces (in this case - a tab is tab with a length of 2 spaces). This is different from indentation of 2 spaces with tab (in this case - a tab is not a tab but 2 normal spaces). I hope the wording is understandable. :D

[the remark this answers is not in the document]
Comment (9inpachi) [reply packages/phoenix-ng/projects/phoenix-ui-components/src/components/ui-menu/collections-info/collections-info.component.ts]: Anyway, we have Prettier now so you can take a look at "[Code formatting](https://github.com/HSF/phoenix/blob/master/CONTRIBUTING.md#code-formatting)",

**Pull request closed without merging**